scenarios/secrets_in_the_cloud/README.md

```diff
@@ -0,0 +1,46 @@
+# Scenario: secrets_in_the_cloud
+
+**Size:** Large
+
+**Difficulty:** Hard
+
+**Command:** `$ ./cloudgoat.py create secrets_in_the_cloud`
+
+## Resources
+
+* 1 Lambda
+* 1 EC2
+* 1 S3
+* 1 DynamoDB
+* 1 Secrets Manager Secret
+* 2 IAM Users
+
+## Start
+
+1. IAM User: low-priv-user
+
+## Goal
+
+Find the final secret in Secrets Manager.
+
+## Summary
+
+As an IAM user with limited privileges, the attacker initiates their journey by examining AWS resources to uncover clues and hidden information. This investigation ultimately results in acquiring a role that grants access to the scenario's primary objective: retrieving the final secret from Secrets Manager. Please note that this walkthrough outlines the intended path for completing the scenario. However, alternative approaches may exist, and we would be delighted to learn about any unique methods used to reach the finish line.
+
+## Path
+
+![Path](./path.png)
+
+## Walkthrough
+
+1. As the IAM user "low-priv-user", the attacker explores the AWS environment, discovering an S3 bucket containing the URL to a web application hosted on an EC2 instance.
+2. The attacker enumerates the web application and uncovers an admin page that prompts for an API key.
+3. While examining the admin page's HTML comments, the attacker finds information about environment variables and a HashiCorp endpoint.
+4. By enumerating the scenario's Lambda function, the attacker locates the web application's API key and uses it to obtain the HashiCorp vault token.
+5. The attacker exploits the vault token to log in to the HashiCorp endpoint and acquires the id_rsa key, which grants SSH access to the EC2 instance hosting the web application.
+6. Intrigued by the login message, the attacker queries the EC2 metadata service and discovers that IMDSv2 is in use.

```

Comment:
Can you add the commands needed to perform these steps in the cheat sheet?

Reply:
I think we decided to skip this one

Reply:
I made some changes to the README walk through and the cheat sheet so they're more descriptive.